docs/functionalabstractions/concretetypes/index.md

```diff
@@ -3,4 +3,111 @@ id: index
 title: "Introduction"
 ---
 
-Coming soon.
+ZIO Prelude features a set of fundamental functional abstractions for working with concrete data types.
+
+These abstractions fall into two categories.
+
+## Properties Of Concrete Types
```

Comment:
I prefer lower cases in titles for small words like of/the. I think it's conventional actually.

Reply:
Adjusted titles.